dev-utils/run-script.js

```diff
@@ -184,7 +193,13 @@ const scripts = {
   runNodeTests,
   runBundleTests,
   buildE2eBundles,
-  startTestServers
+  startTestServers() {
+    let servers = startTestServers.apply(this, arguments)
```

Comment:
We do the same thing on runSauceTests as well, can you move this to the top and use it there as well.